src/store/token/actions.js

```diff
@@ -167,6 +169,60 @@ export function checkCurrentBalance() {
   };
 }
 
+export function checkOtherTokens() {
+  return async (dispatch, getState) => {
+    const { safe } = getState();
+
+    try {
+      const otherTokens = await core.token.listAllTokens(safe.currentAccount);
```

Comment:
add something like
```
.filter((value) -> values > 0.005)
//.sort() not needed because it is sorted when returned from core method
.reverse();
```

Reply:
use core.utils.toFreckles(tcToCrc(Date.now(), Number(0.005))), instead of 0.005 because at this point it s in CRC not TCRC